bin/tessel-2.js

```diff
@@ -73,6 +73,47 @@ parser.command('init')
   })
   .help('Initialize repository for your Tessel project')
 
+parser.command('wifi')
+  .option('list', {
+    abbr: 'l',
+    flag: true,
+    help: "List available Wifi networks"
+  })
+  .option('ip', {
+    abbr: 'i',
+    flag: true,
+    help: "Show Tessel's IP ADDRESS"
+  })
+  .option('ssid', {
+    abbr: 'n',
+    help: "Set the SSID of the network to connect to"
+  })
+  .option('password', {
+    abbr: 'p',
+    help: "Set the password of the network to connect to"
+  })
+  .callback(function(opts) {
+    if (opts.list) {
+      controller.printAvailableNetworks(opts, function(err) {
+        if (err) throw err;
+        process.exit(1);
+      });
+    }
+    else if (opts.ip) {
+      controller.printIPAddress(opts, function(err) {
+        if (err) throw err;
+        process.exit(1);
+      });
+    }
+    else if (opts.ssid && opts.password) {
+      controller.connectToNetwork(opts, function(err) {
+        if (err) throw err;
+        process.exit(1);
+      })
+    }
+  })
+  .help('Configure the wireless connection');
+
```

Comment:
@HarleyKwyn I believe this command uses the same format as the others but it's the only one that has different options for what the command should do based on flags. I still use the `parser.command('wifi')` but I also need to check the flags to figure out which function to call.